extract_subs.py

```diff
@@ -312,7 +350,20 @@ def get_root_dir(path: str) -> str:
         app_run_config = AppRunConfig(target_path=path, target_languages=target_languages,
                                       merge_languages_pairs=merge_languages_pairs,
                                       validation_regex=validation_regex, opensubtitles_auth=opensubtitles_auth,
-                                      download_online=args.download_online)
+                                      download_online=args.download_online, listen_new=args.listen_new)
 
         sub_extract = ExtractSubs(app_run_config, storage)
         sub_extract.scan_files()
+        if args.listen_new:
+            watcher = ListenNewFiles(sub_extract, app_run_config.target_path)
+
+
+            def stop_app(g, i):
+                watcher.stop()
+                logging.info("File listener stopped")
+
+
+            watcher.start()
+            signal.signal(signal.SIGINT, stop_app)
+            signal.signal(signal.SIGTERM, stop_app)
+            signal.pause()
```

Comment:
Rather than handling all signals use [atexit](https://docs.python.org/3/library/atexit.html)

Reply:
Thank, it's exactly what I need. But in this case I replace this peace of code with context manager